apps/designer/package.json

```diff
@@ -9,7 +9,7 @@
     "typecheck": "tsc",
     "lint": "eslint . --ext .ts,.tsx --max-warnings 0",
     "test": "NODE_OPTIONS=--experimental-vm-modules jest",
-    "checks": "yarn lint && yarn typecheck && yarn prettier --check && yarn test",
+    "checks": "pnpm lint && pnpm typecheck && pnpm prettier --check && pnpm test",
```

Comment:
all can be run in parallel as looks independent

Reply:
Yes, will propose some solution